pkg/collector/collector_test.go

```diff
@@ -1,9 +1,14 @@
 package collector
 
 import (
+	"github.com/rs/zerolog"
 	"testing"
 )
 
+func init() {
```

Comment:
does this belong here? Shouldn't this be in main?

Reply:
Moved to another PR